### PR TITLE
port to QGIS3 WIP

### PR DESCRIPTION
**Pull request by NyakudyaA**

### CODE REVIEW

**Comment by NyakudyaA** [comment]
@Samweli can you take a look at this when you have a moment. Still, WIP but the main file that needs attention is `third_party/raven/utils/serializer/base.py` in

File: third_party/raven/utils/serializer/base.py [hunk 1]
```diff
@@ -121,10 +122,10 @@ def serialize(self, value):
 
 
 class LongSerializer(Serializer):
-    types = (long,)
+    types = (int,)
```
Comment (Samweli) [inline third_party/raven/utils/serializer/base.py]: why using int?

**Comment by Samweli** [comment]
@NyakudyaA I tried to load the plugin in the latest QGIS master, got a ModuleNotFoundError: No module named 'third_party'

**Comment by NyakudyaA** [comment]
> @NyakudyaA I tried to load the plugin in the latest QGIS master, got a ModuleNotFoundError: No module named 'third_party'

Hi, @Samweli can you try to use the First Aid plugin and see where the error is coming from.

**Comment by Samweli** [comment]
> Hi, @Samweli can you try to use the First Aid plugin and see where the error is coming from.

Sure, here is the screenshot


third_party_error

**Comment by NyakudyaA** [comment]
This is no longer needed since https://github.com/kartoza/sg-diagram-downloader/pull/49 has been merged

**Pull request closed without merging**